Dockerfile

```diff
@@ -1,3 +1,17 @@
-FROM node:6.4.0-onbuild
-ENV INFOSITE http://shields.io
+FROM node:6.9.2-alpine
```

Comment:
Probably best to keep this at exactly the same version as what's being used in prod. @espadrine what version is currently in use on prod?